modules/virt-adding-vtpm-to-vm.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-vtpm-devices.adoc
+:_content-type: PROCEDURE
+[id="virt-adding-vtpm-to-vm_{context}"]
+= Adding a vTPM device to a virtual machine
+
+Adding a vTPM device to a virtual machine (VM) stores secrets for that VM.
+
+[NOTE]
+====
+vTPM implementation is non-persistent. vTPM discards stored secrets
+after each shutdown.
+====
+
+.Procedure
+
+. Edit the YAML manifest to add a vTPM device.
```

Comment:
Does it need to be the VMI manifest or can it be the VM manifest?

Reply:
Either